install/kubernetes/helm/istio/charts/grafana/values.yaml

```diff
@@ -4,7 +4,7 @@
 enabled: false
 replicaCount: 1
 image:
-  repository: grafana/grafana
+  repository: docker.io/grafana/grafana
```

Comment:
grafana/grafana is not a hub/image combo - its some weird hardcode state of these values.  I'd recommend changing repository to hub, and permitting the new `hub` to be set to `docker.io/grafana` and adding an image name for the image name.  (image.name).  Third party repositories that Istio doesn't manage need to be able to specify both image and hub.

Cheers
-steve

Reply:
@sdake thanks for the feedback.

This goes back to the way I modified it this recently closed PR:
https://github.com/istio/istio/pull/14312

In this PR, @linsun gave us the following feedback:
> Yes, for this PR, I would prefer to keep the old helm values API and add fully qualified image name. Then if you need this change in the new installer, you need a different PR for the installer repo.

I can modify this PR adhere to either approach, I think we just need to agree on which one to use. :-)